src/app/CMakeLists.txt

```diff
@@ -75,18 +75,13 @@ set_source_files_properties(
 )
 
 if (CMAKE_SYSTEM_NAME STREQUAL "Darwin")
-    # substitute @EXECUTABLE@ in dist/mac/Info.plist
-    get_target_property(EXECUTABLE qbt_app OUTPUT_NAME)
-    configure_file(${qBittorrent_SOURCE_DIR}/dist/mac/Info.plist
-        ${qBittorrent_BINARY_DIR}/dist/mac/pregen.plist @ONLY)
-    file(GENERATE
-        OUTPUT ${qBittorrent_BINARY_DIR}/dist/mac/Info.plist
-        INPUT ${qBittorrent_BINARY_DIR}/dist/mac/pregen.plist
-    )
+    # provide variables for substitution in dist/mac/Info.plist
+    get_target_property(EXECUTABLE_NAME qbt_app OUTPUT_NAME)
+    set(MACOSX_DEPLOYMENT_TARGET ${CMAKE_OSX_DEPLOYMENT_TARGET})
```

Comment:
It is probably best to to add a comment here referencing this issue and that the variable name should be changed once we drop `qmake`. Something like `# This variable name should be changed once qmake is no longer used. Refer to the discussion in PR #14813`.

Reply:
suggested comment added